etna/lib/helpers.rb

```diff
@@ -30,6 +30,9 @@ def magma_client
     @magma_client ||= Etna::Clients::Magma.new(
         token: token,
         ignore_ssl: EtnaApp.instance.config(:ignore_ssl),
+        # Persistent connections cause problem with magma restarts, until we can fix that we should force them
```

Comment:
I noticed that using persistent connections to magma while executing model updates has a small chance of failing.  I think what happens is that the persistent connection delays the restart of update_model actions, which can cause future commands to execute before the sequel model has updated.  Seems to go away without persistent connections.

This only effects the etna bin.  In the future we should investigate other solutions.

Reply:
Interesting find -- nice investigative work for this and the add_project fix.